File: Python/Easy/مجموع المصفوفة.py

# Creator: OmarAfet
# https://profile.satr.codes/OmarAfet/public/overview
# https://github.com/OmarAfet

def calculate_sum(lst: list[int]) -> int:
    result = 0

    for i in lst:
        result += i
        if i == 7:
            return 0
        
    return result

File: JavaScript/Easy/مجموع المصفوفة.js
/*

Creator: OmarAfet
https://profile.satr.codes/OmarAfet/public/overview
https://github.com/OmarAfet

*/

function calculate_sum(lst) {
	let sum = 0;
	for (let i = 0; i < lst.length; i++) {
		if (lst[i] == 7) {
			break;
		} else {
			sum += lst[i];
		}
	}
	return sum;
}
